movement.cpp

```diff
@@ -0,0 +1,435 @@
+#include "movement.h"
+#include "pixy.h"
+
+front_left_motor_ = Motor(frontLeftMotorPins[0], frontLeftMotorPins[1], frontLeftPwm);
+front_right_motor_ = Motor(frontRightMotorPins[0], frontRightMotorPins[1], frontRightPwm);
+back_left_motor_ = Motor(backLeftMotorPins[0], backLeftMotorPins[1], backLeftPwm);
+back_Right_motor_ = Motor(backRightMotorPins[0], backRightMotorPins[1], backRightPwm);
+
+void Movements::move0()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.forward();
+  front_right_motor_.forward();
+  back_left_motor_.forward();
+  back_right_motor_.forward();
+}
+
+void Movements::move45()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.forward();
+  front_right_motor_.stop();
+  back_left_motor_.stop();
+  back_right_motor_.forward();
+}
+
+void Movements::move90()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.forward();
+  front_right_motor_.backward();
+  back_left_motor_.backward();
+  back_right_motor_.forward();
+}
+
+void Movements::move135()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.stop();
+  front_right_motor_.backward();
+  back_left_motor_.backward();
+  back_right_motor_.stop();
+}
+
+void Movements::move180()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.backward();
+  front_right_motor_.backward();
+  back_left_motor_.backward();
+  back_right_motor_.backward();
+}
+
+void Movements::move225()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.backward();
+  front_right_motor_.stop();
+  back_left_motor_.stop();
+  back_right_motor_.backward();
+}
+
+void Movements::move270()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.backward();
+  front_right_motor_.forward();
+  back_left_motor_.forward();
+  back_right_motor_.backward();
+}
+
+void Movements::move315()
+{
+  front_left_motor_.changePwm(230.0);
+  front_right_motor_.changePwm(230.0);
+  back_left_motor_.changePwm(230.0);
+  back_right_motor_.changePwm(230.0);
+  front_left_motor_.stop();
+  front_right_motor_.forward();
+  back_left_motor_.forward();
+  back_right_motor_.stop();
+}
+
+void Movements::movement(int angle)
```

Comment:
A better name for this function could be setDirection. It seems very similar to name of the class right now.